### PR TITLE
TFA Fix- Moving the Blue store pinned tests to the baremetal

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
**TFA Error:**

rc = test_mod.run(
File "/home/jenkins/ceph-builds/18.2.0-128/Weekly/rados/17/cephci/tests/rados/test_bluestore_features.py", line 94, in run
mem_growth = is_what_percent_mem(mem_list)
File "/home/jenkins/ceph-builds/18.2.0-128/Weekly/rados/17/cephci/tests/rados/test_bluestore_features.py", line 178, in is_what_percent_mem
min_mem = int(min(mem_list))
ValueError: invalid literal for int() with base 10: ''

**Root cause:**

The method "get_heap_dump" cannot able to get the OSD heap dump due to OSD getting flapped or down sometimes in the rhos-d environment.

Jira story- https://issues.redhat.com/browse/RHCEPHQE-12175

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by SrinivasaBharath** [comment]
Reef Pass Log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-O5Y5WI

File: suites/quincy/baremetal_pipeline/mero_4_node_rados_db_wal_conf.yaml [hunk 1]
```diff
@@ -92,6 +92,66 @@ tests:
...
+  -
+    test:
+      abort-on-fail: true
+      config:
+        command: add
+        copy_admin_keyring: true
+        id: client.1
+        install_packages:
+          - ceph-common
+          - cephadm
```
Comment (pdhiran) [inline suites/quincy/baremetal_pipeline/mero_4_node_rados_db_wal_conf.yaml:104]: we can remove the cephadm package from client.

Comment (SrinivasaBharath) [reply suites/quincy/baremetal_pipeline/mero_4_node_rados_db_wal_conf.yaml]: Removed the cephadm package

**Review by pdhiran — APPROVED** [review]
Looks good.

**Pull request merged**